admin-manual/installation-setup/installation/install-advanced.rst

```diff
@@ -0,0 +1,67 @@
+.. _install-advanced:
+
+=============================
+Advanced installation options
+=============================
+
+*On this page*
+
+* :ref:`Installing for development <development>`
+* :ref:`Installing across multiple machines <multiple-machines>`
+* :ref:`Configuring Archivematica with SSL <SSL-support>`
+
+.. _development:
+
+Install for development
+-----------------------
+
+The recommended way to install Archivematica for development is with Docker.
+For instructions on how to deploy Archivematica in Docker containers, see
+the :ref:`Archivematica Docker repo <docker>`.
+
+
+It's also possible to deploy Archivematica using Ansible and Vagrant. For
+instructions on how to install Archivematica from a virtual machine, see the
+`Ansible & Vagrant Installation instructions <https://wiki.archivematica.org/Getting_started#Installation>`_
+on the Archivematica wiki. See also instructions for installation on a
+virtual machine using Vagrant in the :ref:`Quick Start Guide <quick-start-install>`.
+
+
+.. _multiple-machines:
+
+Installing across multiple machines
+-----------------------------------
+
+It is possible to spread Archivematica's processing load across several machines
+by installing the following services on separate machines:
+
+* Elasticsearch
+* gearman
+* MySQL
+
+For help, send an email to the `archivematica-tech`_ mailing list.
+
+Firewall
+^^^^^^^^
+
+When installing Archivematica on multiple machines, all the machines must be
+able to reach each other on the following ports:
+
+* http, mysqld, gearman, nfs, ssh
```

Comment:
Perhaps we don't need a list if there is only one item, e.g. they could be just an enumeration of inline literals after the colon:

    ... each other on the following ports: ``http``, ``mysqld``, ``gearman``, ``nfs``, ``ssh``.